tests/test_markitdown.py

```diff
@@ -229,8 +230,104 @@ def test_markitdown_exiftool() -> None:
+    result = markitdown.convert(
+        Path(TEST_FILES_DIR) / "test_with_comment.docx",
+        style_map="comment-reference => ",
+    )
+    for test_string in DOCX_COMMENT_TEST_STRINGS:
+        text_content = result.text_content.replace("\\", "")
+        assert test_string in text_content
+
+    # Test DOCX processing, with comments and setting style_map on init
+    markitdown_with_style_map = MarkItDown(style_map="comment-reference => ")
+    result = markitdown_with_style_map.convert(
+        Path(TEST_FILES_DIR) / "test_with_comment.docx"
+    )
+    for test_string in DOCX_COMMENT_TEST_STRINGS:
+        text_content = result.text_content.replace("\\", "")
+        assert test_string in text_content
+
+    # Test PPTX processing
+    result = markitdown.convert(Path(TEST_FILES_DIR) / "test.pptx")
+    for test_string in PPTX_TEST_STRINGS:
+        text_content = result.text_content.replace("\\", "")
+        assert test_string in text_content
+
+    # Test HTML processing
+    result = markitdown.convert(
+        Path(TEST_FILES_DIR) / "test_blog.html", url=BLOG_TEST_URL
+    )
+    for test_string in BLOG_TEST_STRINGS:
+        text_content = result.text_content.replace("\\", "")
+        assert test_string in text_content
+
+    # Test ZIP file processing
+    result = markitdown.convert(Path(TEST_FILES_DIR) / "test_files.zip")
+    for test_string in DOCX_TEST_STRINGS:
+        text_content = result.text_content.replace("\\", "")
+        assert test_string in text_content
+
+    # Test Wikipedia processing
+    result = markitdown.convert(
+        Path(TEST_FILES_DIR) / "test_wikipedia.html", url=WIKIPEDIA_TEST_URL
+    )
+    text_content = result.text_content.replace("\\", "")
+    for test_string in WIKIPEDIA_TEST_EXCLUDES:
+        assert test_string not in text_content
+    for test_string in WIKIPEDIA_TEST_STRINGS:
+        assert test_string in text_content
+
+    # Test Bing processing
+    result = markitdown.convert(
+        Path(TEST_FILES_DIR) / "test_serp.html", url=SERP_TEST_URL
+    )
+    text_content = result.text_content.replace("\\", "")
+    for test_string in SERP_TEST_EXCLUDES:
+        assert test_string not in text_content
+    for test_string in SERP_TEST_STRINGS:
+        assert test_string in text_content
+
+    # Test non-UTF-8 encoding
+    result = markitdown.convert(Path(TEST_FILES_DIR) / "test_mskanji.csv")
+    text_content = result.text_content.replace("\\", "")
+    for test_string in CSV_CP932_TEST_STRINGS:
+        assert test_string in text_content
+
+
+@pytest.mark.skipif(
+    skip_exiftool,
+    reason="do not run if exiftool is not installed",
+)
+def test_markitdown_exiftool_pathlib() -> None:
+    markitdown = MarkItDown()
+
+    # Test JPG metadata processing
+    result = markitdown.convert(Path(TEST_FILES_DIR) / "test.jpg")
+    for key in JPG_TEST_EXIFTOOL:
+        target = f"{key}: {JPG_TEST_EXIFTOOL[key]}"
+        assert target in result.text_content
```

Comment:
the tests seem repetitive. Could you wait until the test refactor PR is approved and merged and add tests for pathlib later?

Reply:
i need to wait for sometime before adding the tests right